<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLColorLegendDisplayNodeWidget</class>
 <widget class="qMRMLWidget" name="qMRMLColorLegendDisplayNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Legend</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/Colors.png</normaloff>../Icons/Colors.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="VisibilityLabel">
     <property name="text">
      <string>Visibility:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ctkCheckBox" name="ColorLegendVisibilityCheckBox">
     <property name="indicatorIcon">
      <iconset resource="../../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc">
       <normaloff>:/Icons/VisibleOn.png</normaloff>
       <normalon>:/Icons/VisibleOff.png</normalon>:/Icons/VisibleOn.png</iconset>
     </property>
     <property name="indicatorIconSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="DisplayedOnViewLabel">
     <property name="text">
      <string>View:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="qMRMLDisplayNodeViewComboBox" name="DisplayNodeViewComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="TitleTextLabel">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="TitleTextLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="LabelTextLabel">
     <property name="text">
      <string>Label text:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="UseScalarValueAsLabelTextRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Show numeric value as label text</string>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">LabelTextButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="UseColorNameAsLabelTextRadioButton">
       <property name="toolTip">
        <string>Show color name as label text</string>
       </property>
       <property name="text">
        <string>Color name</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">LabelTextButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="NumberOfLabelsLabel">
     <property name="text">
      <string>Number of labels:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="NumberOfLabelsSpinBox">
     <property name="toolTip">
      <string>Number of labels to display. Only applicable if values are used as label text (not color name).</string>
     </property>
     <property name="maximum">
      <number>64</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="MaxNumberOfColorsLabel">
     <property name="text">
      <string>Number of colors:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="MaxNumberOfColorsSpinBox">
     <property name="toolTip">
      <string>Maximum number of colors displayed. Reduce the number to see discrete colors instead of a continuous color gradient. Only applicable if values are used as label text (not color name).</string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ColorLegendOrientationLabel">
     <property name="text">
      <string>Orientation:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="VerticalOrientationRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Vertical</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">ColorLegendOrientationButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="HorizontalOrientationRadioButton">
       <property name="text">
        <string>Horizontal</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">ColorLegendOrientationButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="PositionWithinViewLabel">
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Horizontal position of the color legend</string>
       </property>
       <property name="text">
        <string>H:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSlider" name="PositionXSlider">
       <property name="toolTip">
        <string>Horizontal position of the color legend</string>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="pageStep">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Vertical position of the color legend</string>
       </property>
       <property name="text">
        <string>V:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSlider" name="PositionYSlider">
       <property name="toolTip">
        <string>Vertical position of the color legend</string>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="pageStep">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="DimensionsLabel">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>Short side length of the color legend, as percentage of view size</string>
       </property>
       <property name="text">
        <string>S:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSlider" name="ShortSideSizeSlider">
       <property name="toolTip">
        <string>Short side length of the color legend, as fraction of view size</string>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="pageStep">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>Long side length of the color legend, as percentage of view size</string>
       </property>
       <property name="text">
        <string>L:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ctkDoubleSlider" name="LongSideSizeSlider">
       <property name="toolTip">
        <string>Long side length of the color legend, as fraction of view size</string>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="pageStep">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="TitlePropertiesGroupBox">
     <property name="title">
      <string>Title properties</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ctkVTKTextPropertyWidget" name="TitleTextPropertyWidget">
        <property name="showText" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="ctkCollapsibleGroupBox" name="LabelPropertiesGroupBox">
     <property name="title">
      <string>Label properties</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="ctkVTKTextPropertyWidget" name="LabelTextPropertyWidget">
        <property name="textLabel">
         <string>Format:</string>
        </property>
        <property name="showSize" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="ctkDynamicSpacer" name="DynamicSpacer">
     <property name="activeSizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLDisplayNodeViewComboBox</class>
   <extends>qMRMLCheckableNodeComboBox</extends>
   <header>qMRMLDisplayNodeViewComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkVTKTextPropertyWidget</class>
   <extends>QWidget</extends>
   <header>ctkVTKTextPropertyWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSlider</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDynamicSpacer</class>
   <extends>QWidget</extends>
   <header>ctkDynamicSpacer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../Libs/MRML/Widgets/Resources/qMRMLWidgets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qMRMLColorLegendDisplayNodeWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>DisplayNodeViewComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="ColorLegendOrientationButtonGroup"/>
  <buttongroup name="LabelTextButtonGroup"/>
 </buttongroups>
</ui>
